File: CS/GridViewWithDxScrollView/Controllers/HomeController.cs
using DevExpress.Web.Mvc;
using GridViewWithDxScrollView.Models;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.Mvc;

namespace GridViewWithDxScrollView.Controllers
{
    public class HomeController : Controller
    {
        // GET: Home
        public ActionResult Index()
        {
            return View();
        }

        [ValidateInput(false)]
        public ActionResult GridViewPartial() {
            var model = ModelRepository.GetData();
            return PartialView("_GridViewPartial", model);
        }
    }
}

File: VB/GridViewWithDxScrollView/Controllers/HomeController.vb
Imports DevExpress.Web.Mvc
Imports GridViewWithDxScrollView.Models
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Web
Imports System.Web.Mvc

Namespace GridViewWithDxScrollView.Controllers
	Public Class HomeController
		Inherits Controller

		' GET: Home
		Public Function Index() As ActionResult
			Return View()
		End Function

		<ValidateInput(False)>
		Public Function GridViewPartial() As ActionResult
			Dim model = ModelRepository.GetData()
			Return PartialView("_GridViewPartial", model)
		End Function
	End Class
End Namespace